ra2ce/analysis/losses/time_values/time_values_reader.py

```diff
@@ -33,7 +33,9 @@ class TimeValuesReader(LossesInputDataReaderBase):
     """
 
     csv_columns = ["trip_types", "value_of_time", "occupants"]
-    data_type = TimeValues
+
+    def __init__(self) -> None:
+        self.object_type = TimeValues
```

Comment:
it's a dataclass, use a default factory instead (and define types)

Reply:
Done (although for a field of type `type` no factory is needed)